promisasu/Portal
# Bug Fixes

**Taiga User Story:** None

**Taiga Task(s):** None

**What did you change?**
Fixed bug where completed Survey Start and End time would not render correctly.
Improved query for active surveys to more clearly show the date range.
Switch server side code from using `Object.create` to `Object.assign` to avoid setting values to prototype, which cannot be easily viewed/logged.
Bumped `pm2` version with latest bug fixes.

**How can we test?**
* Open the webapp or mobile app for a patient with an active survey (survey should appear).
* Visit a survey page: start, end and completed should appear correctly.
